.styleci.yml

```diff
@@ -1,5 +1,4 @@
 preset: psr2
 
 enabled:
-  - long_array_syntax
```

Comment:
That will let us use the short array suntax. See this comment: https://github.com/KnpLabs/php-github-api/pull/389#discussion_r68899974